package.json

```diff
@@ -8,42 +8,42 @@
   },
   "bugs": "https://github.com/aha-app/aha-cli/issues",
   "dependencies": {
-    "@fal-works/esbuild-plugin-global-externals": "^2.1.1",
-    "@oclif/command": "^1",
-    "@oclif/config": "^1",
-    "@oclif/plugin-help": "^3",
-    "@types/inquirer": "^7.3.1",
-    "chalk": "^4.1.0",
-    "chokidar": "^3.4.3",
-    "cli-ux": "^5.5.1",
-    "crypto": "^1.0.1",
-    "debug": "^4.1.1",
-    "esbuild": "^0.11.2",
-    "form-data": "^3.0.0",
-    "http-call": "^5.3.0",
-    "inquirer": "^8.0.0",
-    "netrc-parser": "^3.1.6",
-    "open": "^7.3.0",
-    "tslib": "^1"
+    "@fal-works/esbuild-plugin-global-externals": "2.1.1",
+    "@oclif/command": "1.6.1",
+    "@oclif/config": "1.15.1",
+    "@oclif/plugin-help": "3.1.0",
```

Comment:
would it be possible to go up to latest on this? https://github.com/oclif/plugin-help/pull/305 which I believe was released in one of the more recent versions removes a potentially insecure `lodash.template` transitive dependency. Looks like there was also a recent release to address the recursive dep issue.